src/lib/types/apiResponses.d.ts

```diff
@@ -167,7 +167,7 @@ export interface PublicMerchItem {
   uuid: UUID;
   itemName: string;
   collection?: PublicMerchCollection;
-  picture: string;
+  picture: string | null;
```

Comment:
For some reason, "no option item" in the Fall 2001 collection has `picture` set to `null`. I've updated the type here to reflect that, but it instead might be an issue with the backend. I'm guessing to allow for multiple images, this will be replaced with an array later on